Bruteforce docs (#1657)
nextcloud/documentation: admin_manual/configuration_server/bruteforce_configuration.rst

======================
Brute force protection
======================

> Nextcloud has build in protection against brute force attempts. This protects

Reviewer: ```suggestion
Nextcloud has built-in protection against brute force attempts. This protects
```

your system from attackers trying for example a lot of different passwords.

Brute force protection is enabled by default on Nextcloud.

How it works
------------

The brute force protection is easiest to see in action at the login page.

> If you try to login the first time with an invalid username and/or password you

Reviewer: ```suggestion
If you try to log in the first time with an invalid username and/or password you
```

will not notice anything. But if you do this a few times you start to notice
that the verification of the login is taking longer each time. This is the
brute force protection kicking in.

The maximum delay is 25 seconds.

After a successfull login the attempts will be cleared. And once a user is
properly authenticated they will not longer be hit by the delay.

Troubleshooting
---------------

On most setups Nextcloud will work out of the box without any issues. If you
run into a situation where login is often very slow for all users the first
step is to inspect the `bruteforce_attempts` table. There you can see

> which ip addresses are actually throttled.

Reviewer: ```suggestion
which IP addresses are actually throttled.
```
